configure.ac

```diff
@@ -0,0 +1,38 @@
+AC_PREREQ([2.69])
+AC_INIT([dyz], [0.1], [https://github.com/Igalia/dyz],
+        [dyz], [https://github.com/Igalia/dyz])
+AC_CONFIG_HEADER([src/config.h:src/config.h.in])
```

Comment:
To end passing the path where the lua files will be installed on the dyz.c program.

I think the whole point of using lua is being able to edit the lua files files in place without having to recompile the whole thing.